tkhomich/08_jquery/Python_Example.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html> 
+<HTML> 
+<HEAD>
+	<META charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+	<TITLE>Программа на Python — часть 1</TITLE>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.css">
+
+</HEAD> 
+<BODY>
+
+		<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```

Comment:
Skeleton does not have these classes

Reply:
this is example file. not related tp homework. I'll remove it.